### DISCUSSION TITLE
Why downloading microsoft fonts from sourceforge when building livecd ISO?

### DISCUSSION THREAD

**Discussion by CicadaSeventeen** [SpiralLinux/SpiralLinux-project#295] [General]
Hello, I have been making my fork of SpirialLinux for some time.
During building, I find that `live-build` try to download and then install Microsoft fonts ( Windows fonts I guess ) from sourceforge. However, I cannot see such codes in the hooks of SpirialLinux source.
I do not think it a good idea to use Windows fonts in Linux. Do you have any idea about what makes `live-build` do so? Do you know how to skip it?

**Comment by CicadaSeventeen** [comment]
@geckolinux  Do you have any idea?

**Comment by methanoid** [comment]
Look at all the contents of the packages which install fonts? ;-)    BTW email me (gmail) if you're willing to discuss 1-2-1 your DebianEasy?

**Comment by geckolinux** [comment]
Hi there, that's from the `ttf-mscorefonts-installer` package.
